games-action/polymc/polymc-1.4.0.ebuild

```diff
@@ -65,6 +67,8 @@ COMMON_DEPENDS="
 
 DEPEND="
 	${COMMON_DEPENDS}
+	app-text/scdoc
+	kde-frameworks/extra-cmake-modules
```

Comment:
I have a hunch that both of these should be BDEPEND instead. Are we executing them during building or do we just need them present? Most of the packages depending on these two declare them as BDEPEND.